src/model/trust.page.model.ts

```diff
@@ -0,0 +1,27 @@
+type TrustDetails = {
```

Comment:
This seems to be a different way of modelling this to other bits of the model, e.g.
beneficial.owner.individual.model.ts
https://github.com/companieshouse/overseas-entities-web/blob/main/src/model/beneficial.owner.individual.model.ts

Please could you make it more like that?

Reply:
By which I mean, explicit separate keys, rather than implicit.

Reply:
After talking with @markpit & @mwestacott, we're OK with this explicit mapping model as-is, provided that there is a comment in any explicit mappers noting that they behave differently to the mapping elsewhere in the app.